### PR TITLE
chore: update models

### PR DESCRIPTION
**Pull request by hahuyhoang411**
## Describe Your Changes

- [x] Stable Zephyr 3B
- [x] Delete Yarn Mistral (issue: https://github.com/janhq/jan/issues/1549)
- [x] Dolphin Phi-2 (https://github.com/janhq/jan/pull/1603)
- [x] LlamaCorn-1.1B (https://huggingface.co/jan-hq/LlamaCorn-1.1B)
- [x] Update new version for OpenChat-3.5 7B
- [x] Update model-extension's version

## Self Checklist

- [x] Added relevant comments, esp in complex areas)
- [x] Created issues for follow-up changes or refactoring needed

### CODE REVIEW

File: models/dolphin-phi-2/model.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ],
+    "id": "dolphin-phi-2",
+    "object": "model",
+    "name": "Dolphin Phi-2 2.7B Q8",
+    "version": "1.0",
+    "description": "Dolphin Phi-2 is a 2.7B model, fine-tuned for chat, excelling in common sense and logical reasoning benchmarks.",
+    "format": "gguf",
+    "settings": {
+      "ctx_len": 4096,
+      "prompt_template": "<|im_start|>system\n{system_message}<|im_end|>\n<|im_start|>user\n{prompt}<|im_end|>\n<|im_start|>assistant\n",
```
Comment (louis-jan) [inline models/dolphin-phi-2/model.json:16]: Could I ask why there is an extra '\n' at the end?

File: models/llamacorn-1.1b/model.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    ],
+    "id": "llamacorn-1.1b",
+    "object": "model",
+    "name": "LlamaCorn 1.1B Q8",
+    "version": "1.0",
+    "description": "LlamaCorn is designed to improve chat functionality from TinyLlama.",
+    "format": "gguf",
+    "settings": {
+      "ctx_len": 2048,
+      "prompt_template": "<|im_start|>system\n{system_message}<|im_end|>\n<|im_start|>user\n{prompt}<|im_end|>\n<|im_start|>assistant",
```
Comment (louis-jan) [inline models/llamacorn-1.1b/model.json:16]: Does this model require prompt template?

File: models/dolphin-phi-2/model.json (hunk 1 above)
Comment (0xSage) [reply models/dolphin-phi-2/model.json]: @hahuyhoang411 can we resolve these comments and get this pr merged or closed? Thanks!

**Review by louis-jan — APPROVED** [review]
LGTM

**Pull request merged**